diagnostic.md

```diff
@@ -6,70 +6,77 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+API="${API_ORIGIN:-http://localhost:4741}"
+URL_PATH="/books/${ID}"
+curl "${API}${URL_PATH}" \
+  --include \
+  --request GET
+
+echo
+
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+scripts/books/show.sh
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
-```
+bundle exec rake runs the version of rake named in the gem file which solves dependencies
+db:migrate updates a schema by adding, removing, or updating tables, columns, or entries```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+scaffold is a full set (model, database migration, controller, views, and test suite)
+controller is just setting up the controller
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+describe which attributes and relationships should be serialized
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```md
-// your response here
+it will return a list of all patients in the database
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
 ```md
-// your response here
+models/patient.rb
 ```
 
 Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
-```
+model access the database based on information from the controller```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
```

Comment:
This is a typo and should have been `bin/rails console` so no points will be deducted for this.